index.html

```diff
@@ -2,10 +2,85 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Hello World</title>
+    <title>Apple</title>
+    <link rel="stylesheet" href="assets/css/styles.css"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous"/>
+    <link rel="icon" href="assets/img/icon.png"/>
   </head>
   <body>
-    <h1>Hello World</h1>
-    <p>This is a web page.</p>
+    <main>
+      <div class="container">
+        <div>
+          <nav class="gradient">
+            <ul class="nav_ul flex_container">
+              <li id="apple" class="nav_hover"><a class="nav_link white" href=""><img src="assets/img/apple_logo.png" width="40px" height="auto"/></a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">Store</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">Mac</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iPod</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iPhone</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iPad</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iTunes</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">Support</a></li>
+              <li><input class="gradient" type="search" name="search"/></li>
+            </ul>
+          </nav>
+        </div>
+
+        <div class="content">
+          <h2>What will your verse be?</h2>
+          <p>
+            Everyone has something to share. See how people are using iPad in their unique ways to add to the world&#39;s story.
+          </p>
+          <span><a href="" class="body_link">Watch the &quot;Your Verse&quot; film and explore the stories <i id="play_icon" class="far fa-play-circle"></i></a></span>
+        </div>
+      </div>
+    </main>
+   
+    <section class="promo">
+      <div class="flex_container">
+        <figure>
+          <a href=""><img src="assets\img\promo_iphone5s.jpg" alt=""/></a>
+        </figure>
+
+        <figure>
+          <a href=""><img src="assets\img\promo_iphone5c.jpg" alt=""/></a>
```

Comment:
For this section 
- [ ] Each link is an <a> HTML tag, with a <div> tag inside it.
- [ ] The <div> tag  inside the <a> tag should use an image as background 

- [ ] The text on the original page is actually an image